src/gui/screen_menu_statistics.cpp

```diff
@@ -19,3 +22,9 @@ void ScreenMenuStatistics::windowEvent(window_t *sender, GUI_event_t event, void
 
     ScreenMenu::windowEvent(sender, event, param);
 }
+
+void ScreenMenuStatistics::updateUptime() {
+    char buffer[32];
+    snprintf(buffer, sizeof(buffer), "Uptime: %lu hours", ticks_s() / 3600);
+    uptime_label.SetText(buffer);
```

Comment:
1. SetText takes `string_view_utf8`, you're providing char[]
2. We have separate fields for "text" and for "value" - putting everything together is not acceptable
3. This code does not take translations into account
4. The label should at least also show minutes as well